templates/csv.yaml

```diff
@@ -322,7 +322,7 @@ spec:
               containers:
                 - name: knative-openshift
                   # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-operator
+                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:knative-operator
```

Comment:
Can't we use variable from project.yaml here?

Reply:
yep - will do something for that on master